apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -32,6 +34,7 @@ mod port;
 const APP_NAME: &str = "vx-mark-scan-controller-daemon";
 const KPB_200_FW_VID: u16 = 0x28cd;
 const KPB_200_FW_PID: u16 = 0x4008;
+const STARTUP_SLEEP_DURATION: Duration = Duration::from_millis(1500);
```

Comment:
Curious how much extra padding this 1.5 second sleep includes. Just want to make sure this sleep is sufficient to resolve the problem every time. And am wondering if there's any harm in bumping up just a bit more, e.g., to 3 seconds?

Reply:
Since we saw the daemon initialize correctly on startup 1/3 times, I'm guessing the race condition is <100ms. But the risk of increasing to 3s is pretty low, so went ahead and did that.